### ISSUE TITLE
Access to boot and reset buttons

### ISSUE THREAD

**Issue by briped** [Chrismettal/flipper-zero-backpacks#4]
A small suggestion for any future iterations of the case. Add holes/openings in the case. over the boot and reset buttons, so it's possible to press them without having to disassemble the board from the case.

I'm going to find a small drill and make some holes myself, but seeing as you've made a small hole to see the LED through, two more for the buttons would probably be simple for you to make (not me, I couldn't CAD if my life depended on it).

Thanks for a nice product :)

**Comment by Chrismettal** [comment]
Good Idea. I guess we could integrate a button extension into the case without adding another part to the BOM.
